src/main/java/com/mraof/minestuck/block/MSBlocks.java

```diff
@@ -193,7 +225,22 @@ public static void registerBlocks(final RegistryEvent.Register<Block> event)
 		registry.register(new Block(Block.Properties.create(Material.EARTH, MaterialColor.GRAY).hardnessAndResistance(0.5F).harvestTool(ToolType.SHOVEL).sound(SoundType.GROUND)).setRegistryName("dark_gray_chess_dirt"));
 		registry.register(new Block(Block.Properties.create(Material.EARTH, MaterialColor.LIGHT_GRAY).hardnessAndResistance(0.5F).harvestTool(ToolType.SHOVEL).sound(SoundType.GROUND)).setRegistryName("light_gray_chess_dirt"));
 		registry.register(new SkaiaPortalBlock(Block.Properties.create(Material.PORTAL, MaterialColor.CYAN).doesNotBlockMovement().lightValue(11).hardnessAndResistance(-1.0F, 3600000.0F).noDrops()).setRegistryName("skaia_portal"));
-		
+
+		Block goldBricks = register(registry, new Block(Block.Properties.create(Material.ROCK, MaterialColor.YELLOW).hardnessAndResistance(6.0F, 9.0F).harvestTool(ToolType.PICKAXE).lightValue(13).harvestLevel(3)).setRegistryName("glowing_gold_bricks"));
+		registry.register(new ModStairsBlock(goldBricks.getDefaultState(), Block.Properties.from(goldBricks)).setRegistryName("glowing_gold_brick_stairs"));
+		registry.register(new SlabBlock(Block.Properties.from(goldBricks)).setRegistryName("glowing_gold_brick_slab"));
+		registry.register(new Block(Block.Properties.from(goldBricks)).setRegistryName("glowing_gold_brick_chiseled"));
```

Comment:
Oh, just noticed that you're calling it "brick chiseled" here when "chiseled" is normally a prefix (you do call it "chiseled gold bricks" in translation, so I think `chiseled_glowing_gold_bricks` is better)
Also applies to the purple variant